### PR TITLE
Remove unneeded using statements

### PR DESCRIPTION
**Pull request by salmonz**
Removed as per suggestion in https://github.com/Microsoft/msbuild/issues/21

### CODE REVIEW

File: src/Shared/InternalErrorException.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 using System;
 using System.Diagnostics;
-using System.Security.Permissions; // for SecurityPermissionAttribute
+// for SecurityPermissionAttribute
```
Comment (dannyvv) [inline src/Shared/InternalErrorException.cs]: Would you mind yanking this line as well :) It doesn't make sense anymore :)

**Comment by dannyvv** [comment]
Looks good to me barring the comment for SecurityPermissionAttribute

**Comment by AndyGerlicher** [comment]
Closing as duplicate of #31. There's a PR for this functionality in CodeFormatter right now. When that gets in, we can use it to format all the code now (and in the future).

**Comment by akoeplinger** [comment]
CodeFormatter PR for reference: https://github.com/dotnet/codeformatter/pull/101

**Comment by sean-gilliam** [comment]
I closed the issue #21 so hopefully no further PRs get submitted between now and the CodeFormatter PR gets included.

**Pull request closed without merging**